packages/design-preview/src/components/style-variation.tsx

```diff
@@ -26,7 +33,15 @@ const StyleVariationPreview: React.FC< StyleVariationPreviewProps > = ( {
 	}, [ variation, base ] );
 
 	return (
-		<div className="design-preview__style-variation" tabIndex={ 0 } role="button">
+		<div
+			className={ classnames( 'design-preview__style-variation', {
+				'design-preview__style-variation--is-active': isActive,
+			} ) }
+			tabIndex={ 0 }
+			role="button"
+			onClick={ () => onClick?.( variation ) }
+			onKeyDown={ ( e ) => e.keyCode === SPACE_BAR_KEYCODE && onClick?.( variation ) }
```

Comment:
I can't seem to make use of this, how do I use the space key to "click" a variation other than the current one without using the mouse?

Reply:
This was an a11y linting error, it's meant to used when a preview button is focused via tab, and users can press the space bar to select it.